timer-selector.js

```diff
@@ -479,74 +528,125 @@ function buildTimer(language) {
     data.styling,
   );
 
-  outputElement.value = buildWikitext(data.template, url, data.height, data.width);
+  outputElement.value = buildWikitext(language, data.template, url, score, data.height, data.width);
+
+  if (copyToClipboard) {
+    navigator.clipboard.writeText(outputElement.value);
+  }
+}
+
+function setMessage(language, id, messageKey = null) {
+  document.getElementById(id).innerText = getMessage(language, messageKey || id);
+}
+
+function initializeSummaryDeletionMessages(language) {
+  // Summary deletion reasons vary by site
+  var summaryDeletionBox = document.getElementById('summary-deletion-reason');
+  var messages = getMessage(language, 'summary-deletion-reasons', true);
+  for (const message of messages) {
```

Comment:
We go to pains to only use old-style JS for compatible support everywhere else, it doesn't make sense to require something new just here.
```suggestion
  for (var i = 0; i < messages.length; i++) {
    var message = messages[i];
```